scripts/newinstall.sh

```diff
@@ -187,11 +187,12 @@ fi
 
 if true; then
 	# Check the version by running a small Python program (taken from the Python EUPS package)
+    # XXX should not assume that python is in the default path
```

Comment:
The comment isn't relevant to `$PYTHON`, it is about the check being a fatal error if python isn't in the path.  The logic needs to be refactored.